### ISSUE TITLE
Minimap Coverage

### ISSUE THREAD

**Issue by knfk** [arianne/stendhal#439]
Problem

When my character sits on a small map, e.i. "0_semos_city", the minimap shows the whole map, so I can click in any position and my character will make a planned move to that spot using some path finding algorithm. But when the map in question is bigger, for example, "0_semos_plain_n", which is 4 times as big, the minimap will only represent 1/4 of it, so attempts to cross the map through planned movements would be severely restrained. I also can't have a good overview of the region map my character is located at.

tldr: I can't view the whole map in the minimap (when in big maps), so I can't make planned movements across it.

Solutions

1 - Increase the size of the minimap so that it can natively represent the biggest maps available in their entirety
2 - Zoom feature that allows players to focus on parts of the current map, or zoom out to show the whole thing

**Comment by nhnb** [comment]
The minimap only showing the area around you is by design.
